client/src/components/EmailInput/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import InputField from "../../elements/InputField";
+
+const EmailInput = () => (
```

Comment:
You can directly use InputField in the Form, instead of creating a email input field, same thing for pwd

Reply:
I did that so I can use the same Email and Password input field in Sign up form as well. Let me know if it's a bad idea

Reply:
And It will also prevent me or any team member from passing down incorrect props

Reply:
Its fine :)